pkg/cluster/cache_whitebox_test.go

```diff
@@ -175,6 +175,25 @@ func TestGetClustersByType(t *testing.T) {
 			assert.Contains(t, clusters, member2)
 		})
 
+		t.Run("found after refreshing the cache", func(t *testing.T) {
+			// given
+			defer resetClusterCache()
+			member := newTestFedCluster("member", Member, ready)
+			called := false
+			clusterCache.refreshCache = func() {
```

Comment:
should you also reset the `clusterCache.refreshCache` in `resetClusterCache()` ?

Reply:
this is done by `defer resetClusterCache()` - check the function https://github.com/codeready-toolchain/toolchain-common/blob/21d2a6b78d83291625f7dcc54cca5b26f6bf954f/pkg/cluster/cache_whitebox_test.go#L420-L422
it creates a brand new instance of the `clusterCache`